resources/views/components/quantity.blade.php

```diff
@@ -17,6 +18,7 @@ class="shrink-0 pl-2.5 text disabled:cursor-not-allowed disabled:opacity-50"
             name="qty"
             type="number"
             dusk="qty"
+            value="1"
```

Comment:
I think we don't need a value here.

Reply:
We don't but this is so we don't have any changes after JS has loaded, before it was just empty before JS loaded
its just a small thing that prevents any visual changes after JS has loaded